components/clp-rust-utils/src/package_config/config.rs

```diff
@@ -0,0 +1,87 @@
+use serde::Deserialize;
+
+#[derive(Deserialize, Clone)]
+pub struct Config {
+    #[serde(default)]
+    pub package: Package,
+    #[serde(default)]
+    pub database: Database,
+    #[serde(default)]
+    pub results_cache: ResultsCache,
+    #[serde(default)]
+    pub api_server: ApiServer,
+}
+
```

Comment:
```suggestion
/// Mirror of `clp_py_utils.clp_config.Database`.
///
/// # NOTE
///
/// * This type is partially defined: unused fields are omitted and discarded through
///   deserialization.
/// * The default values must be kept in sync with the Python definition.
```
Showing it as an example. If it looks good to u, we can apply it to other types as well.